docs/strategy.md

```diff
@@ -0,0 +1,9 @@
+# k8gb strategy options
+## failover
+Pinned to a specified primary cluster until workload on that cluster has no available Pods, upon which the next available cluster's Ingress node IPs will be resolved. When Pods are again available on the primary cluster, the primary cluster will once again be the only eligible cluster for which cluster Ingress node IPs will be resolved
+
+## roundRobin
```

Comment:
Please move it to the first place to be consistent with https://github.com/k8gb-io/k8gb/pull/556/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R145

Reply:
It was lost during rebase, now it is there